04week/tomorrows-technology/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE HTML>
+<html>
+<link rel="stylesheet" href="css/style.css" />
+
+<body>
+  <header>
+
+  </header>
+  <main>
+    <h1>TOP STORIES</h1>
```

Comment:
The top stories section does not move along as the page is resized, and seems to overlap with the background image